hocr-check

```diff
@@ -57,9 +57,17 @@ def mostly_nonoverlapping(boxes,significant_overlap=0.2):
 ### main
 ################################################################
 
-if len(sys.argv)<1:
+def print_usage():
     print "usage:",sys.argv[0],"[-o] file.html"
+
```

Comment:
I tried to stay as true to the current source as possible, not changing the output or coding style. But yes, it would be good to improve it once we have a baseline (all scripts printing help information, all scripts free off PyXML and unit tests running).